docs/Installing.asciidoc

```diff
@@ -190,6 +191,11 @@ You can install the main openQA server package using these commands.
 # openSUSE
 zypper in openQA
 
+# SLE
+. /etc/os-release
+SUSEConnect -p sle-module-desktop-applications/$VERSION_ID/$CPU
```

Comment:
Likely we also need
```
SUSEConnect -p sle-module-development-tools/$VERSION_ID/$CPU
```
there. At least some Perl packages comeing from there are missing in @perlpunk 's VM otherwise. (In my container this repo was already there but supposedly this is not always the case.)

Reply:
Yes, I confirmed. I had everything installed but had development tools dependencies fulfilled from an IBS internal repo. Double-checked that now.